bundles/org.openhab.binding.modbus.helioseasycontrols/README.md

```diff
@@ -0,0 +1,306 @@
+| sensorConfigKwlFtf1              | general         | Sensor configuration (installed or not) KWL-FTF 1                | Switch                   |      | OFF = no sensor, ON = sensor installed                                                               |
+| sensorConfigKwlFtf2              | general         | Sensor configuration (installed or not) KWL-FTF 2                | Switch                   |      | OFF = no sensor, ON = sensor installed                                                               |
+| sensorConfigKwlFtf3              | general         | Sensor configuration (installed or not) KWL-FTF 3                | Switch                   |      | OFF = no sensor, ON = sensor installed                                                               |
+| sensorConfigKwlFtf4              | humidityControl | Sensor configuration (installed or not) KWL-FTF 4                | Switch                   |      | OFF = no sensor, ON = sensor installed                                                               |
+| sensorConfigKwlFtf5              | humidityControl | Sensor configuration (installed or not) KWL-FTF 5                | Switch                   |      | OFF = no sensor, ON = sensor installed                                                               |
+| sensorConfigKwlFtf6              | humidityControl | Sensor configuration (installed or not) KWL-FTF 6                | Switch                   |      | OFF = no sensor, ON = sensor installed                                                               |
+| sensorConfigKwlFtf7              | humidityControl | Sensor configuration (installed or not) KWL-FTF 7                | Switch                   |      | OFF = no sensor, ON = sensor installed                                                               |
+| sensorConfigKwlFtf8              | humidityControl | Sensor configuration (installed or not) KWL-FTF 8                | Switch                   |      | OFF = no sensor, ON = sensor installed                                                               |
+| globalManualWebUpdate            | humidityControl | Manual update of data (profile 8LXGP, XWP, firmware, SD files)   | Switch                   |      | ON = start update                                                                                    |
+| portalGlobalsErrorForWeb         | humidityControl | Always the latest error that has occurred                        | Number                   |      | 1 - 255                                                                                              |
+| clearError                       | humidityControl | Clear error (PortalGlobals.ErrorForWeb)                          | Switch                   |      | ON = reset error                                                                                     |

```

Comment:
Action?

Reply:
Probably - but wouldn't it mean that instead of defining a Switch with a mapping of only `ON` the user would have to implement also a script to trigger the action?